.github/workflows/check.yaml

```diff
@@ -147,4 +147,25 @@ jobs:
         with:
           name: test-results
           path: testresults--all-features.json
-          
\ No newline at end of file
+
+  doctest:
+    # Run doctests separately, as nextest doesn't yet (https://github.com/nextest-rs/nextest/issues/16)
+    outputs:
+      test_results_url: ${{ steps.doctest_results.outputs.artifact-url }}
```

Comment:
any reason why this is not added to the workflow's outputs (as the output from nextest)?

Reply:
Yeah - it's already been to long since I did all this; fixed.